src/js/helpers/nodes/nodesHelper.js

```diff
@@ -90,3 +91,16 @@ const parseFragments = (root) => findBoundaries(root)
   .map(({ id, nodes }) => createFragment(id, nodes));
 
 export const findFragmentsInContent = () => parseFragments(document.body);
+
+const createVisItem = ({ debug }) => ({
+  id: debug.fragment.id,
+  content: debug.fragment.id,
+  start: debug.startTime,
+  end: debug.finishTime,
+  group: 1,
+});
+
+export const constructFragmentsTimeline = (fragments) => ({
+  items: new DataSet(fragments.map((fragment) => createVisItem(fragment))),
```

Comment:
You can also use it like this 
`fragments.map(createVisItem)`
This will apply arguments automatically to this function